### ISSUE TITLE
Incorrect gesture recognition

### ISSUE THREAD

**Issue by cong-or** [anasmorahhib/3D-CNN-Gesture-recognition#6]
Hi,

I trained the model with a validation subset and training subset using the provided scripts.

The training samples are ~ 1GB and the validation ~ 150 MB.

I trained the model with 10 epochs and the accuracy output was >98%.

However, when running test-app.py the results are not correct. I.e none of my gestures are predicted correctly.

Is there something I may have omitted?

P.S thanks for the code, it's v clean compared to other 3DD-CNN implementations.

**Comment by anasmorahhib** [comment]
Hello,
if you train locally, try to decrease the number of classes to detect that way you will have more data on each class.
you mentioned that you have an output of precision> 98%, but you did not mention if it is a precision of validation or training.
if the training accuracy is more than 98% maybe you have overfitting.
you must also draw the two curves of the validation and training accuracy to see more information and also the epochs when the model falls into overfitting if it was the case.

**Comment by cong-or** [comment]
Thanks @anasmorahhib 

I obtained much better results on remote GPUS. There was an anomaly with my local setup.

Appreciate the help!
